game/p4ss/resource/ui/hudspeedo.res

```diff
@@ -0,0 +1,108 @@
+"Resource/UI/HudSpeedo.res"
+{
+	"HudSpeedo"
+	{
+		"fieldName"		"HudSpeedo"
+		"visible"		"1"
+		"enabled"		"1"
+		"xpos"			"cs-0.5"	[$WIN32]
+		"ypos"			"r150"	[$WIN32]
+		"zpos"			"1"
+		"wide"			"110"
+		"tall"			"50"
+
+	}	
+
+	"background"
+	{
+		"ControlName"	"CTFImagePanel"
+		"fieldName"		"background"
+		"xpos"			"12"
+		"ypos"			"6"
+		"zpos"			"0"
+		"wide"			"76"
+		"wide_minmode"	"56"
+		"tall"			"38"
+		"visible"		"0"
+		"visible_minmode"	"0"
+		"enabled"		"0"
+		"image"			"../hud/misc_ammo_area_blue"
+		"scaleImage"	"1"	
+		"teambg_2"		"../hud/misc_ammo_area_red"
+		"teambg_2_lodef"	"../hud/misc_ammo_area_red_lodef"
+		"teambg_3"		"../hud/misc_ammo_area_blue"
+		"teambg_3_lodef"	"../hud/misc_ammo_area_blue_lodef"				
```

Comment:
Extra whitespace at the end of this line and elsewhere on other lines in the file.